examples/user_guide/Continuous_Coordinates.ipynb

```diff
@@ -36,11 +36,12 @@
    "source": [
     "import numpy as np\n",
     "import holoviews as hv\n",
-    "hv.extension('matplotlib')\n",
+    "from holoviews import opts\n",
+    "\n",
+    "hv.extension('bokeh')\n",
     "\n",
     "np.set_printoptions(precision=2, linewidth=80)\n",
-    "hv.Dimension.type_formatters[np.float64] = '%.2f'\n",
-    "%opts HeatMap (cmap=\"hot\")"
+    "opts.defaults(opts.HeatMap(cmap='fire'), opts.Layout(shared_axes=False))"
```

Comment:
`opts.defaults` should general be formatted like `.opts` with entries on newlines (when you have more than one)

Reply:
I'm don't think that's sensible for one or two keywords defaults.